chainbench/main.py

```diff
@@ -44,19 +48,68 @@ def cli(ctx: click.Context):
     ctx.obj = ContextData()
 
 
+def validate_method(ctx, param, value) -> str:
```

Comment:
I suggest either put type annotations everywhere and run mypy or not put them at all to make code look consistent.

My personal recommendation to use type annotations everywhere to make code more readable, easy to refactor and less error prone due to static analysis.

Reply:
you're referring to the type annotation for arguments to this method right?

Reply:
I have made updates to the codebase. Pls re-review. thanks!